elastic/kibana
# [Serverless] Disable Advanced settings plugin

Partially addresses https://github.com/elastic/kibana/issues/159590

## Summary

This PR disables the Advanced settings plugin for all projects in serverless.

**How to test:**

1. Start Elasticsearch with `yarn es snapshot` and Kibana with `yarn serverless-{mode}` where {mode} can be `es`, `security`, or `oblt`.
2. Verify that the Advanced settings app is not accessible and its endpoint (`app/management/kibana/settings`) leads to the Stack Management landing page.
